### ISSUE TITLE
CI fails in 10.2 branch

### ISSUE THREAD

**Issue by phil-davis** [owncloud/docs#1172]
Make a simple PR to 10.2 branch - e.g. #1170 
CI fails with: https://drone.owncloud.com/owncloud/docs/3190/5
```
Invalid Xrefs Detected:

worktree: /drone/src | component: server | version: 10.2
  path: modules/ROOT/pages/index.adoc | xref: master@admin_manual:appliance/index.adoc

antora: xref validation failed! See previous report for details.
```

**Comment by tboerger** [comment]
The 10.2 branch have not been tested at all until today.

**Comment by phil-davis** [comment]
Then the problem has likely been waiting there for a while. I will diff between master and 10.2 branches and see what "backport" had some missing part.

**Comment by tboerger** [comment]
That's the downside if people who branch off don't understand the build and test process ;)

**Comment by phil-davis** [comment]
Fixed by #1182
